src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--day-of-mounth--28">
+      <div class="calendar__day calendar__day--first-day--sun"></div>
```

Comment:
According instruction this modifier have to be in block `calendar `
![image](https://user-images.githubusercontent.com/119425265/207404888-cb3b45d3-6005-4465-a54a-b01c11b27cd5.png)

Reply:
I think it's not for calendar class

Reply:
you can keep it here, but I think it would be better to move the first-day--sun modifier to the calendar